src/team-extension.ts

```diff
@@ -141,10 +141,18 @@ export class TeamExtension  {
 
     //Opens the file history page for the currently active file
     public OpenFileHistory(): void {
-        //TODO: Add History support for TFVC
-        if (this._manager.EnsureInitialized(RepositoryType.GIT)) {
-            if (this._gitClient) {
+        if (this._manager.EnsureInitialized(RepositoryType.ANY)) {
+            if (this._manager.RepoContext.Type === RepositoryType.GIT && this._gitClient) {
                 this._gitClient.OpenFileHistory(this._manager.RepoContext);
+            } else if (this._manager.RepoContext.Type === RepositoryType.TFVC) {
+                let editor = window.activeTextEditor;
```

Comment:
consider whether this should simply be "this._manager.Tfvc.TfvcViewHistory()" and all the logic (under the TFVC else branch) be inside there. This would just keep this file a bit smaller.